src/main/java/io/cdap/plugin/http/sink/batch/HTTPRecordWriter.java

```diff
@@ -52,109 +64,62 @@
  */
 public class HTTPRecordWriter extends RecordWriter<StructuredRecord, StructuredRecord> {
   private static final Logger LOG = LoggerFactory.getLogger(HTTPRecordWriter.class);
+  private static final String REGEX_HASHED_VAR = "#s*(\\w+)";
```

Comment:
Why do we need `s*` in this regex ? any specific reason ?

I think  `#(\\w+)` should work fine.

Reply:
The PUT or DELETE URL will look something like this https://customer-url/user/#user_id
Tested this regex `#s*(\\w+)` against this input: `/#user_id`, on a web site: _regexr.com_ and it validated successfully. But for the one suggested by you: `#(\\w+)`, the input validation failed.

Reply:
<img width="360" alt="Screenshot 2023-10-20 at 12 12 39 PM" src="https://github.com/data-integrations/http/assets/86019177/1177bf0c-6b3f-413a-bd31-c4ea1b909b54">
<img width="359" alt="Screenshot 2023-10-20 at 12 12 30 PM" src="https://github.com/data-integrations/http/assets/86019177/2da77d14-c2fa-425c-9e09-96dbc5810b44">

Reply:
I think it should work. Can you recheck ?

Reply:
Yes, it does. Replaced with this `#(\\w+)`